src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-druid-integration.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Apache Druid integration
+tags:
+  - Druid integration
+  - apache druid integration
+  - New Relic integrations
+metaDescription: Use New Relic infrastructure agent to get a dashboard with druid metrics.
+freshnessValidatedDate: 2024-04-21
+---
+
+import infrastructureDruidDashboard from 'images/infrastructure_screenshot-full_apache-druid-dashboard.webp'
+
+Integrating Apache Druid with New Relic enhances your ability to monitor, analyze, and optimize the performance of your Druid clusters. By leveraging New Relic's powerful monitoring and alerting features, you can ensure the reliability and efficiency of your Druid-based analytics platform.
+
+<img
```

Comment:
I know fig captions are under debate, but i think a simple pathing caption might be good here. if only to add a little negative space padding around the path